sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -3357,7 +3431,7 @@
       "id": "ecabef"
     },
     {
-      "html": "<p><span style=\"background-color:rgb(255,255,255);color:rgba(0,0,0,0.87);\">By continuing into the StudentAidBC Application you agree to StudentAid BC’s Privacy policy.</span></p>",
+      "html": "<p><span style=\"background-color:rgb(255,255,255);color:rgba(0,0,0,0.87);\">By continuing into the StudentAidBC Application you agree to StudentAid BC&rsquo;s Privacy policy.</span></p>\n",
```

Comment:
what is `&rsquo;`

Reply:
BC&rsquo;s  =>  BC's

Reply:
did you add that? we can directly use `BC's` instead of HTML Alphanumeric value